xgbserver/dummy_pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.poetry]
+name = "workaround-for-editable-install"
+version = "0.0.1"
+description = ""
+authors = ["none"]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.12"
```

Comment:
Should we have a [comment like in other ROCKs](https://github.com/canonical/kserve-rocks/blob/main/pmmlserver/dummy_pyproject.toml#L8)?